libs/thread.liq

```diff
@@ -10,23 +10,25 @@ def thread.run(~fast=true, ~delay=0., ~every=float_getter(-1.), f)
     ignore(f() == ())
     every()
   end
-  thread.run.recurrent(fast=fast, delay=delay, f)
+  thread.run_recurrent(fast=fast, delay=delay, f)
 end
 
+let thread.run.recurrent = thread.run_recurrent
```

Comment:
Could we avoid having the function defined in two places in the `thread` module?

Reply:
IF you can, don't think that should be a blocker.